flair/data.py

```diff
@@ -282,11 +283,11 @@ class DataPoint:
     def __init__(self) -> None:
         self.annotation_layers: Dict[str, List[Label]] = {}
         self._embeddings: Dict[str, torch.Tensor] = {}
-        self._metadata: Dict[str, typing.Any] = {}
+        self._metadata: Dict[str, Any] = {}
 
     @property
     @abstractmethod
-    def embedding(self):
+    def embedding(self) -> None:
```

Comment:
The `embedding` property returns an embedding tensor. All classes that inherit from `DataPoint` (such as `Sentence`) are embeddable and should thus be able to return an embedding tensor.

A snippet to show the property in action: 

```python
from flair.data import Sentence
from flair.embeddings import TransformerDocumentEmbeddings

embeddings = TransformerDocumentEmbeddings("distilbert-base-uncased")

sentence = Sentence("hello")

embeddings.embed(sentence)

# print the embedding of the data point
print(sentence.embedding)
```